packages/orbit-components/src/Card/Card.stories.tsx

```diff
@@ -12,113 +13,118 @@ import { ELEMENT_OPTIONS } from "../Heading/consts";
 
 import Card from ".";
 
-export default {
-  title: "Card",
+type CardPropsAndCustomArgs = React.ComponentProps<typeof Card> & {
+  expanded: boolean;
+  sectionTitle: string;
+  sectionDescription: string;
+  initialExpanded?: boolean;
 };
 
-export const Default = ({ title, titleAs }) => {
-  return <Card title={title} titleAs={titleAs} />;
-};
+const meta: Meta<CardPropsAndCustomArgs> = {
+  title: "Card",
+  component: Card,
+
+  args: {
+    description: "This is description of the card",
+    title: "Card with title",
+    titleAs: ELEMENT_OPTIONS.H2,
+    sectionTitle: "Section Title",
+    sectionDescription: "Section Description",
+    expanded: false,
+    initialExpanded: false,
+    dataA11ySection: "",
+    loading: false,
+  },
 
-Default.args = {
-  title: "Card with title",
-  titleAs: ELEMENT_OPTIONS.H2,
-};
+  argTypes: {
+    titleAs: {
+      options: Object.values(ELEMENT_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
+  },
 
-Default.argTypes = {
-  titleAs: {
-    options: Object.values(ELEMENT_OPTIONS),
-    control: {
-      type: "select",
+  parameters: {
+    controls: {
+      exclude: ["labelClose"],
     },
   },
 };
 
-export const CardWithDescription = ({ title, description }) => {
-  return <Card onClose={action("onClose")} title={title} description={description} />;
-};
+export default meta;
 
-CardWithDescription.story = {
-  name: "Card with description",
+type Story = StoryObj<CardPropsAndCustomArgs>;
+
+export const Default: Story = {
+  render: args => <Card {...args} />,
+  args: {
+    sectionTitle: undefined,
+    sectionDescription: undefined,
+  },
+  parameters: {
+    controls: {
+      exclude: ["labelClose", "sectionTitle", "sectionDescription"],
```

Comment:
Hm, shouldn't we also exclude expanded, initialExpanded, dataA11ySection and loading? I don't think they should be in the Default story, no?